php/PhpJsonApiSamples/CreateCustomerJson.php

```diff
@@ -67,10 +67,11 @@ function buildTransaction($oauth_token){
 
 function buildRequestData(){
     //you can assign the values from any input source fields instead of hard coded values.
-    //customer_id represents a value that you would like use as a reference ID to store customer data
+    //customer_id represents a value that you would like use as a reference ID to that customer from your DB/any data storage  
     //This Id can be used for future reference to the particular customer at PayTrace Vault.
```

Comment:
We save our merchant's customer's credit card data in our DB and let the merchant use this information by just passing the customer ID. so it is known as PayTrace vault for that merchant